x-pack/plugins/siem/public/components/stat_items/areachart.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { pure } from 'recompose';
+import { EuiSeriesChart, EuiAreaSeries } from '@elastic/eui/lib/experimental';
+import { AreaChartData, WrappedByAutoSizer, ChartOverlay } from '.';
+import { AutoSizer } from '../auto_sizer';
+
+const ChartBaseComponent = pure<{
+  data: AreaChartData[];
+  width: number | undefined;
+  height: number | undefined;
+}>(({ data, ...chartConfigs }) =>
+  chartConfigs.width && chartConfigs.height ? (
+    <EuiSeriesChart showDefaultAxis={false} {...chartConfigs}>
+      {data.map(series =>
+        series.value != null ? (
+          /**
+           * Placing ts-ignore here for fillOpacity
+           * */
+          // @ts-ignore
+          <EuiAreaSeries
+            key={`stat-items-areachart-${series.key}`}
+            name={`stat-items-areachart-${series.key}`}
+            data={series.value}
+            fillOpacity={0.04}
+            color={series.color}
+          />
+        ) : null
+      )}
+    </EuiSeriesChart>
+  ) : null
+);
+
+export const AreaChart = pure<{ areaChart: AreaChartData[] }>(({ areaChart }) => (
+  <AutoSizer detectAnyWindowResize={false} content>
+    {({ measureRef, content: { height, width } }) => (
+      <WrappedByAutoSizer data-test-subj="wrapped-by-auto-sizer" innerRef={measureRef}>
+        <ChartBaseComponent height={height} width={width} data={areaChart} />
+        <ChartOverlay />
```

Comment:
There are some limitations for EUI chart. It doesn't accept string as x-axis, only numbers are allowed.
Don't want user to interact with the chart to trigger the tooltips, which shows the value for x-axis, is a bit confusing as it is timestamp.